### PR TITLE
Add methods in nsstring category

### PR DESCRIPTION
**Pull request by pratvick**

### CODE REVIEW

File: TDTChocolate/FoundationAdditions/NSString+TDTAdditions.m [hunk 1]
```diff
@@ -13,6 +13,10 @@ + (NSString *)tdt_stringWithUnsignedInteger:(NSUInteger)unsignedInteger {
   return [self stringWithFormat:@"%llu", (unsigned long long)unsignedInteger];
 }
 
++ (NSString *)tdt_stringWithUnsignedLongLong:(uint64_t)unsignedLongLong {
```
Comment (mx4492) [inline TDTChocolate/FoundationAdditions/NSString+TDTAdditions.m]: The method names say "unsigned long long". The implementations works for "unsigned long long". Shouldn't the type also be "unsigned long long" instead of "uint64_t"?

If you want methods for "uint64_t", then the names of the methods should say so.

**Comment by mx4492** [comment]
It should be easy enough to write tests for this, including boundary cases.

**Comment by mx4492** [comment]
GTM

**Comment by ayushgoel** [comment]
@pratvick Please merge this.
